src/utils/debug/texture-selector-utils.spec.ts

```diff
@@ -78,6 +78,45 @@ describe("Texture Selector Utils - selectDebugTextureForTile", () => {
     expect(tile.content.material.pbrMetallicRoughness.baseColorTexture.texture.source.image).toStrictEqual('Test texture');
   });
 
+  test("Should skip adding original texture to userData if already exist and apply debug texture", () => {
+    const tile = {
+      userData: {
+        originalTexture: {
+          material: {
+            pbrMetallicRoughness: {
+              baseColorTexture: {
+                texture: {
+                  source: {
+                    image: "Testing Image",
+                  },
+                },
+              },
+            },
+            texture: "Test texture",
+          },
+        },
+      },
+      content: {
+        material: {
+          pbrMetallicRoughness: {
+            baseColorTexture: {
+              texture: {
+                source: {
+                  image: "Testing Image",
+                },
+              },
+            },
+          },
+          texture: "Test texture",
+        },
+      },
+    };
+    const uvDebugTexture = "Test texture";
+    selectDebugTextureForTile(tile, uvDebugTexture);
+    expect(tile.userData.originalTexture.material.pbrMetallicRoughness.baseColorTexture.texture.source.image).toStrictEqual("Testing Image");
```

Comment:
Format is needed?

Reply:
I am using prettier for auto-formatting and changes are formatted by prettier in the texture-selector-utils.ts. 
But test file texture-selector-utils.spec.ts is not initially formatted so I didn't use prettier also just to have my changes visible for review. Otherwise a lot of extra changes will appear. 
So, just confirm if it needs to update texture-selector-utils.spec.ts by prettier also